_gsocorgs/2021/uclouvain.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Universite Catholique de Louvain"
+author: "Olivier Mattelaer"
+layout: default
+organization: UCLouvain
+logo: uclouvain-logo.jpg
+description: |
+The Université catholique de Louvain is Belgium's largest French-speaking university. It is located in Louvain-la-Neuve.
```

Comment:
Add 2 spaces at the beginning of "The Universite...", otherwise, we may get a YAML exception

Reply:
done